client/i18n/locales/english/intro.json

```diff
@@ -2037,7 +2037,10 @@
         ]
       },
       "uglc": { "title": "199", "intro": [] },
-      "vonu": { "title": "200", "intro": [] },
+      "lab-football-team-cards": {
+        "title": "Build Football Team Cards",
+        "intro": ["", ""]
```

Comment:
we will need an intro here.
you can probably borrow from what is already in the JS beta portion and modify to suit the needs of this lab